vocode/streaming/output_device/twilio_output_device.py

```diff
@@ -55,6 +56,20 @@ def interrupt(self):
     def enqueue_mark_message(self, mark_message: MarkMessage):
         self._mark_message_queue.put_nowait(mark_message)
 
+    def send_dtmf_tones(self, keypad_entries: List[KeypadEntry]):
+        tone_generator = DTMFToneGenerator()
+        for keypad_entry in keypad_entries:
+            logger.info(f"Sending DTMF tone {keypad_entry.value}")
+            dtmf_tone = tone_generator.generate(
+                keypad_entry, sampling_rate=self.sampling_rate, audio_encoding=self.audio_encoding
+            )
+            dtmf_message = {
+                "event": "media",
+                "streamSid": self.stream_sid,
+                "media": {"payload": base64.b64encode(dtmf_tone).decode("utf-8")},
+            }
+            self._twilio_events_queue.put_nowait(json.dumps(dtmf_message))
```

Comment:
Can these get interrupted? And if so is a mark necessary as well, so using `_send_audio_chunk_and_mark()`

Reply:
these cannot get interrupted (and thus mirrors the vonage implementation). if we send an event to `_twilio_events_queue` we can expect that it gets sent to the ws unless the output device gets torn down (when the conversation ends)